### PR TITLE
Doc proofreading. NFC

### PR DESCRIPTION
**Pull request by RequireSun**

### CODE REVIEW

File: site/source/docs/api_reference/preamble.js.rst
Comment (kripken) [inline site/source/docs/api_reference/preamble.js.rst:168]: I think this was intentional, actually. `[a, b[` indicates that a is included in the range, but not b. Sometimes it's written `[a, b)`.

But I agree this is not clear enough. Perhaps we could add text to explain this?

**Review by kripken — COMMENTED** [review]
Thanks! Changes look good aside from one of them that I commented on.

File: site/source/docs/api_reference/preamble.js.rst
Comment (sbc100) [reply site/source/docs/api_reference/preamble.js.rst:168]: `[a, b)` makes more sense to me. How about we switch to that?

File: site/source/docs/tools_reference/emsdk.rst
Comment (sbc100) [inline site/source/docs/tools_reference/emsdk.rst:108]: actually since this is python code lest use spaces about both assignment operators maybe?

File: site/source/docs/api_reference/module.rst
Comment (sbc100) [inline site/source/docs/api_reference/module.rst:122]: While you are there can you drop the `=1` here?

File: site/source/docs/tools_reference/emsdk.rst
Comment (RequireSun) [reply site/source/docs/tools_reference/emsdk.rst:108]: oh, i mistook this file is a shell, so that i remove the space around assignment operator, i'll revert it.

File: site/source/docs/api_reference/module.rst
Comment (RequireSun) [reply site/source/docs/api_reference/module.rst:122]: oh, my fault, i'll remove it.

**Review by sbc100 — APPROVED** [review]

File: site/source/docs/api_reference/module.rst
Comment (RequireSun) [reply site/source/docs/api_reference/module.rst:122]: i've just proofreading the syntax of doc, i havent use WebGL in my project, so that i've no experience about this option

Comment (sbc100) [reply site/source/docs/api_reference/module.rst:122]: No worries.  All options are implicitly `=1` these days so `=1` is always redundant.

**Comment by RequireSun** [comment]
Hello, it's the first time i contribute to this repository, there's an awaiting approval named 'archive'.
Is that means I need to archive the repository i forked in my namespace, so that the workflow will be approval?
I've archived my repository, but the workflow doesnt approval, did I misunderstand it?

**Comment by RequireSun** [comment]
oh, I found that it's a github action which needs anothor maintainer's approval to emit.
sorry for disturbed

**Review by kripken — APPROVED** [review]
Thanks!

**Pull request merged**
